department-of-veterans-affairs/caseflow
# Queue | Add new Fiduciary issues

Resolves #6796 

### Description
Adds new Fiduciary issues to `VACOLS::Issref` table and `ISSUE_INFO.json`

### Acceptance Criteria
- [ ] 4 new Fiduciary issues appear in FE dropdowns
- [ ] `VACOLS::Issref.where(prog_code: "12").count == 5`

### Testing Plan
1. `rake db:seed`
2. Confirm `Issref` table updated
3. Visit `/queue` as attorney or judge, go to case details, draft / review decision flow, edit issue
4. Select Program: Fiduciary, confirm 5 options appear in Issue dropdown.
5. Update issue and save

<details><summary>updated Fiduciary codes</summary>

```diff
diff --git a/client/constants/ISSUE_INFO.json b/client/constants/ISSUE_INFO.json
index bd9678e96..396294823 100644
--- a/client/constants/ISSUE_INFO.json
+++ b/client/constants/ISSUE_INFO.json
@@ -1075,6 +1075,22 @@
       "01": {
         "description": "Fiduciary Appointment",
         "plain_description": "Fiduciary Appointment"
+      },
+      "02": {
+        "description": "Hub Manager removal of a fiduciary under §13.500",
+        "plain_description": "Hub Manager removal of a fiduciary under §13.500"
+      },
+      "03": {
+        "description": "Hub Manager misuse determination under §13.400",
+        "plain_description": "Hub Manager misuse determination under §13.400"
+      },
+      "04": {
+        "description": "RO Director decision upon recon of a misuse determ",
+        "plain_description": "RO Director decision upon recon of a misuse determ"
+      },
+      "05": {
+        "description": "Dir of PFS negligence determination for reissuance",
+        "plain_description": "Dir of PFS negligence determination for reissuance"
       }
     }
   }
```

</details>

<details><summary>tabular</summary>

| Program        | Issue           | Level 1  |
|:------------ |:------------ |:------- |
| 12 | 02 | Hub Manager removal of a fiduciary under §13.500 |
| 12 | 03 | Hub Manager misuse determination under §13.400 |
| 12 | 04 |  RO Director decision upon recon of a misuse determ |
| 12 | 05 | Dir of PFS negligence determination for reissuance |

</details>
<details>
<summary>screenshot</summary>
<img src="https://user-images.githubusercontent.com/8533004/45161363-b2b3e400-b1b9-11e8-8135-875646da5a97.png">
</details>
